### PR TITLE
199

### PR DESCRIPTION
**Pull request by Abigail770**
- Added map filters to sidebar in filter.component.ts.  I used Event Emitters to pass values from the filter component to the map component.  **If you could take a look at my code in filter.component.ts, that would be great!  There may be a better way in Angular of handling interactions between the filter and map component than what I have set up.**
- Added map filters button for mobile screen sizes (max-width 875px).  The map filter accordions are hidden on mobile.  When the map filters button is clicked, the map and filter results are hidden and the map filters are shown.  On submit or clear, the filters are hidden and map and results are shown.
- Desktop vs. mobile layout updates when the window is resized.
- I placed the map filters in an accordion.  When one dropdown filter is opened, any others that are open are collapsed.
- I added a media query to set the min-width of the map container to be less than a mobile screen width, to avoid overflowing the html container width (min-width is still at 550px on desktop sizes).  

**I noticed a couple other issues on mobile that I didn't address:**
-Some of the header elements are overflowing the width of the html container on mobile
-legend and Minimize Map button overlap on mobile
I can create a new issue for these (or two separate issues), whichever you would prefer.

### CODE REVIEW

**Comment by lprivette** [comment]
Oops sorry I forgot to review this yesterday! Is it ready to be reviewed/merged @Abigail770 ?

**Comment by Abigail770** [comment]
Now it's ready!  If you could look over the changes to the code when you get a chance too, that would be great (filter.component.ts and html, map.component.ts and html).  Everything is functioning as it should, but I just want to make sure I'm following best coding practices, etc. since I'm new to Angular.

File: src/app/map/map.component.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ import { FilterResultsComponent } from '@app/filter-results/filter-results.compo
 export class MapComponent implements OnInit {
     @ViewChild(FilterResultsComponent)
     filterResultsComponent: FilterResultsComponent;
-    public removable = true;
+    // public removable = true;
```
Comment (lprivette) [inline src/app/map/map.component.ts]: If this isn't being used lets go ahead an remove it.

If you'd like to leave it in for some future reason just add an explanation comment.

**Review by lprivette — CHANGES_REQUESTED** [review]
Looks good! I actually haven't used EventEmitters before, but they're part of the angular core so that's good enough for me. You could have used an obsverable/service but I don't think one choice is better than the other for this.

Once you address that one commented out variable, I'll go ahead and merge this into dev.

**Review by lprivette — APPROVED** [review]

**Pull request merged**